incubator/openebs/templates/cm-openebs-prometheus-config.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.policies.monitoring.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "openebs.fullname" . }}-prometheus-config
+  namespace: {{ .Values.rbac.namespace }}
```

Comment:
You should probably remove the namespace from the resources. Also, I don't see how this is related to RBAC. The namespace can be specified by the user when they install the chart.

Reply:
this file is removed.